Index: src/practice1/Credit.java
===================================================================
diff --git a/src/practice1/Credit.java b/src/practice1/Credit.java
--- a/src/practice1/Credit.java	
+++ b/src/practice1/Credit.java	
@@ -19,12 +19,11 @@
         // Цикл розраховує розмір щомісячного платежу за тілом і відсотки
         for (int i = 1; i <= period; i++) {
             System.out.println("Сума оплати тіла кредиту за місяць " + i + ": " + getFormat(bodyPay));
-            float percentpay = creditSum * mounthlyPay; // Сума сплати відсотків у місяць
-            System.out.println("Сума оплати відсотків за місяць " + i + ": " + getFormat(percentpay));
-            creditSum = creditSum - bodyPay; // Зміна розміру тіла кредиту після сплати ануїтетного платежу
-            allInterestPay = allInterestPay + percentpay; // Підрахунок загальної суми сплачених відсотків
+            float percentPay = creditSum * mounthlyPay;
+            System.out.println("Сума оплати відсотків за місяць " + i + ": " + getFormat(percentPay));
+            creditSum = creditSum - bodyPay;
+            allInterestPay = allInterestPay + percentPay;
         }
-        // Вивід на екран загальної суми сплачених відсотків
         System.out.println("Загальна сума відсотків: " + getFormat(allInterestPay));
 
         //АНУЇТЕТНИЙ МЕТОД РОЗРАХУНКУ
